xml/Microsoft.VisualBasic.CompilerServices/StringType.xml

```diff
@@ -377,8 +377,8 @@
         <Parameter Name="Value" Type="System.Double" />
       </Parameters>
       <Docs>
-        <param name="Value">Required. <see langword="Double" /> to convert to a <see langword="String" /> value.</param>
-        <summary>Returns a <see langword="String" /> value that corresponds to a specified <see langword="Double" />.</summary>
+        <param name="Value">Required. <see cref="T:System.Double" /> to convert to a <see langword="String" /> value.</param>
+        <summary>Returns a <see langword="String" /> value that corresponds to a specified <see cref="T:System.Double" />.</summary>
```

Comment:
Same, natural language prefered.
Also, langword should not be used for String. It's also a type, so you should use cref=T:System.String.

```suggestion
        <summary>Returns a string value that corresponds to a specified <see cref="T:System.Double" />.</summary>
```